### DISCUSSION TITLE
Access on Mobile

### DISCUSSION THREAD

**Discussion by beanstalkgleams** [TriliumNext/Trilium#2939] [Q&A] [answered]
I started using trilium recently and I'm trying to use it on my phone after installing it on my computer and setting up the server. I know this is a noob question, but how exactly am I supposed to access it on mobile? Is there a website I'm supposed to be using? I tried following the server installation page but I'm really bad with computers and I had a hard time following it.

**Comment by sigaloid** [comment, marked as answer]
You can access the server by finding the computer's IP address, the server port, then entering it into your browser on your mobile device.

**Reply by beanstalkgleams** [reply]
I tried that, but on my phone the site said it wouldn't connect.

**Reply by sigaloid** [reply]
Are they connected to the same network? And have you tried visiting localhost:[port] on your computer as well? If that doesn't work the server isn't running.

**Reply by beanstalkgleams** [reply]
Yeah they're on the same network. When I visit localhost:[port] on my computer it does work, and when I test a sync on trilium desktop or on localhost:[port] on my computer I get confirmation that the sync worked.

**Reply by sigaloid** [reply]
What if you try the same IP you are entering on your phone, on the desktop? You can't use localhost on your phone but you can use the regular IP on your desktop

**Reply by beanstalkgleams** [reply]
It still isn't working. 

Thank you for the help but I've decided I'm just going to use something easier for me to use.
